### PR TITLE
Issue forms: redux

### PR DESCRIPTION
**Pull request by ArtBlue**
Modifications based on additional feedback.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 1]
```diff
@@ -39,6 +25,20 @@ body:
     description: A concise description of what you expected to happen.
   validations:
     required: false
```
[the remark this answers is not in the document]
Comment (ArtBlue) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml:27]: I believe the current default is `false` so, at the moment, they're probably not needed. However, whenever possible, I like being explicit; possibly redundant with the typical assumption, but IMO additional info is better than no info.

**Review by LuLaValva — APPROVED** [review]

**Review by agliga — COMMENTED** [review]
Good set of changes! Thanks for doing this.
Just a couple of minor comments. Otherwise LGTM

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name: 🐞 Bug report
 description: File a bug/issue
-title: "[BUG] <title>"
+title: "<component name>: <issue title>"
 labels: ["type: bug", "Needs Triage"]
```
Comment (ArtBlue) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:4]: @agliga , @ianmcburnie , I'm assuming the `type: bug` and `type:feature` (in the other file) are good initial tags for each respectively. Is `Needs Triage` ok as well for both or do we want some other tag to indicate the new issue need to be reviewed by us?

Comment (ianmcburnie) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml:4]: I think "status: needs triage" or "status: untriaged" would be good as a sensible default.

Comment (ArtBlue) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml]: Thanks! Updated. @agliga , @ianmcburnie , if there's nothing else can you please approve this to indicate you're good with it getting merged?

**Review by agliga — APPROVED** [review]
I think in a follow up we should split up feature to be enhancement 
But i think this is good for now
Thanks for doing this.

**Review by ianmcburnie — APPROVED** [review]

**Pull request merged**